crates/ruma-client-api/src/r0/profile/set_avatar_url.rs

```diff
@@ -75,7 +90,7 @@ mod tests {
                     .uri("https://bar.org/_matrix/client/r0/profile/@foo:bar.org/avatar_url")
                     .body(&[] as &[u8]).unwrap(),
             ).unwrap(),
-            IncomingRequest { user_id, avatar_url: None } if user_id == "@foo:bar.org"
+            IncomingRequest { user_id, avatar_url: None, ..} if user_id == "@foo:bar.org"
```

Comment:
rustfmt doesn't know how to format all macro contents unfortunately. Can you add a space between `..` and `}` please?